release-1.8/release_notes_draft.md

```diff
@@ -50,6 +50,19 @@ promotion indicates RBAC's production readiness and adoption. Today, the
 authorizer is turned on by default by many distributions of Kubernetes, and is a
 fundamental aspect of a secure cluster.
 
+### SIG Node
+
+[SIG Node][] is responsible for the components which support the controlled 
+interactions between pods and host resources as well as managing the lifecycle
+of pods scheduled on a node. For the 1.8 release SIG Node continued to focus
+on supporting the broadest set of workload types, including hardware sensitive
```

Comment:
s/hardware sensitive/performance sensitive - there is a blog coming out that uses that language.

Reply:
or "including hardware and performance sensitive workloads"